Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/DirectorySymlinkTests.cs

```diff
@@ -202,9 +202,10 @@ private IEnumerable<Operation> GetLayoutProducingOperations(string rootDir)
                 .Split(new[] { '\n', '\r' }, StringSplitOptions.RemoveEmptyEntries)
                 .Select(spec => $"{rootDir}/{spec.Trim()}")
                 .Select(spec =>
-                    spec.Contains("->") ? OpCreateSym(X(spec)) :
-                    spec.EndsWith("/")  ? OpCreateDir(X(spec)) :
-                                          OpWriteFile(X(spec)));
+                    spec.Contains("->") ? (spec.EndsWith("/") ? OpCreateSym(X(spec), Operation.SymbolicLinkFlag.DIRECTORY ) : 

```

Comment:
can you please move this `spec.EndsWith("/")` conditional inside the `OpCreateSym` method invocation
```suggestion
                    spec.Contains("->") ? OpCreateSym(X(spec), spec.EndsWith("/") ?Operation.SymbolicLinkFlag.DIRECTORY : Operation.SymbolicFlag.FILE) : 
```